src/data/general-cs-quiz.ts

```diff
@@ -1908,6 +1908,15 @@ const generalCSQuiz = [
     Distractor3: "The ability to create multiple methods with the same name but different implementations",
     Explanation: "Polymorphism is an object-oriented programming principle that allows objects of different classes to be treated as objects of a common super class. It enables a single method to behave differently based on the object that it is acting upon.",
     Link: "https://www.freecodecamp.org/news/a-guide-to-object-oriented-programming-principles/"
+  },
+  {
+    Question: "Which one is NOT a type of computer virus?",
+    Answer: "Micro virus",
+    Distractor1: "Ransomware",
+    Distractor2: "Worm",
+    Distractor3: "Trojan horse",
+    Explanation: "A computer virus is a type of malware that, when executed, replicates itself by modifying other computer programs and inserting its own code into those programs.",
+    Link: "https://en.wikipedia.org/wiki/Computer_virus"
```

Comment:
Please move this question to the security quiz instead

Reply:
Alright, I will be doing that